src/UportSubprovider.js

```diff
@@ -164,4 +164,8 @@ class UportSubprovider {
   }
 }
 
+function encodeSignature({r,s,v}) {
```

Comment:
export this function and use the import in the test to reduce the risk of the two implementations getting out of sync